internal/buildscripts/packaging/msi/make.ps1

```diff
@@ -65,10 +69,8 @@ function Confirm-MSI {
     # start service
     Start-Service otelcol
 
-    # uninstall msi, validate service is uninstalled
+    # uninstall msi
     Start-Process -Wait msiexec "/x `"$msipath`" /qn"
-    sc.exe query state=all | findstr "otelcol" | Out-Null
-    if ($LASTEXITCODE -ne 1) { Throw "otelcol service failed to uninstall" }
```

Comment:
The uninstall validation had to be removed because on GitHub Actions an exit code 1 will immediately fail the workflow. This test attempts to query an uninstalled service to verify it is in fact uninstalled and handles the error.